test/library/standard/Endian/Endian.chpl

```diff
@@ -38,10 +38,10 @@ module Endian {
   private extern proc le32toh(x: c_uint): c_uint;
   private extern proc htobe32(x: c_uint): c_uint;
   private extern proc be32toh(x: c_uint): c_uint;
-  private extern proc htole64(x: c_ulong): c_ulong;
-  private extern proc le64toh(x: c_ulong): c_ulong;
-  private extern proc htobe64(x: c_ulong): c_ulong;
-  private extern proc be64toh(x: c_ulong): c_ulong;
+  private extern proc htole64(x: c_ulonglong): c_ulonglong;
```

Comment:
It seems to me that since these functions include the bit width and always work with unsigned, you should use `uint(32)` `uint(64)` etc.

Reply:
Yeah, it makes sense to me. I have made the changes in 7d728ae

Reply:
Does it still function on your system?

Reply:
Yes, all the tests pass on my system.